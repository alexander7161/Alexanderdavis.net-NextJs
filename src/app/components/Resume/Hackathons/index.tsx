import Hackathons from "./Hackathons";
export default Hackathons;
