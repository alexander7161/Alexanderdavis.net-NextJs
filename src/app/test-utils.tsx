import React, { ReactNode } from "react";
import { render, RenderOptions } from "@testing-library/react";
import { Provider } from "react-redux";
import { Router } from "react-router";
import { makeStore } from "./store";
import ThemeProvider from "./Providers/ThemeProvider";
import { createBrowserHistory } from "history";

const history = createBrowserHistory();

const AllTheProviders = ({ children }: { children?: ReactNode }) => {
	return (
		<Router history={history}>
			<Provider store={makeStore({})}>
				<ThemeProvider>{children}</ThemeProvider>
			</Provider>
		</Router>
	);
};

const customRender = (
	ui: React.ReactElement,
	options?: Omit<RenderOptions, "queries">
) => render(ui, { wrapper: AllTheProviders, ...options });

// re-export everything
export * from "@testing-library/react";

// override render method
export { customRender as render };
