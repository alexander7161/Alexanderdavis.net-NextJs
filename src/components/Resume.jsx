import React, { Component } from "react";
import ReactGA from "react-ga";
import { withStyles } from "@material-ui/core/styles";
import Typography from "@material-ui/core/Typography";

const styles = theme => ({
	root: {
		overflowY: "scroll",
		height: `calc(100vh - ${theme.mixins.toolbar["minHeight"]}px - 72px)`
	}
});

function recordPageView(page) {
	ReactGA.pageview(page);
}

class Resume extends Component {
	componentDidMount() {
		if (this.props.location.pathname === "/resume") {
			recordPageView(this.props.location.pathname);
		} else if (this.props.location.pathname === "/") {
			recordPageView("/");
		}
	}
	componentDidUpdate(prevProps) {
		if (
			this.props.location.pathname === "/resume" &&
			this.props.location.pathname !== prevProps.location.pathname
		) {
			recordPageView(this.props.location.pathname);
		} else if (
			this.props.location.pathname === "/" &&
			this.props.location.pathname !== prevProps.location.pathname
		) {
			recordPageView("/");
		}
	}
	render() {
		return (
			<div
				style={{
					width: "100%",
					marginTop: "25vh",
					textAlign: "center"
				}}
			>
				<Typography variant="title" color="inherit">
					Coming Soon
				</Typography>
			</div>
		);
	}
}

export default withStyles(styles)(Resume);
