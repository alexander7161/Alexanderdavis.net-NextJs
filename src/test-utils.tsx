import React, { ReactNode } from 'react'
import { render } from '@testing-library/react'
import { Provider } from "react-redux";
import { BrowserRouter as Router } from "react-router-dom";
import store from "./configureStore";
import ThemeProvider from "./Providers/ThemeProvider"

const AllTheProviders = ({ children } : {children:ReactNode}) => {
  return (
    <Router>
		<Provider store={store}>
			<ThemeProvider>
        {children}
        </ThemeProvider>
		</Provider>
	</Router>
  )
}

const customRender = (ui, options?) =>
  render(ui, { wrapper: AllTheProviders, ...options })

// re-export everything
export * from '@testing-library/react'

// override render method
export { customRender as render }