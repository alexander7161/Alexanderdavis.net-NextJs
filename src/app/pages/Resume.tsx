import React from "react";
import { NextPage } from "next";
import AppBar from "../components/AppBar";
import Container from "../components/Container";
import { Route } from "react-router";
import BottomTabs from "../components/BottomTabs";
import SwipeableRoutes from "react-swipeable-routes";
import Projects from "../components/projects";
import Resume from "../components/Resume";

const Index: NextPage = () => {
	return (
		<Container>
			<AppBar />
			<SwipeableRoutes
				style={{
					WebkitOverflowScrolling: "touch", // iOS momentum scrolling
					overflowY: "hidden",
				}}
			>
				<Route path="/" component={Projects} />
				<Route exact={true} path="/resume" component={Resume} />
			</SwipeableRoutes>
			<BottomTabs />
		</Container>
	);
};

export default Index;
