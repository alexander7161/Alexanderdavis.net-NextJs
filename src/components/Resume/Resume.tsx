import React from "react";
import ContainerDiv from "../ContainerDiv";
import Hackathons from "./Hackathons";

export default () => {
	return (
		<ContainerDiv>
			<Hackathons />
		</ContainerDiv>
	);
};
