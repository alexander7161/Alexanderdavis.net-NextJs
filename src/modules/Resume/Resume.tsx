import React from "react";
import SwipeablePageContainer from "../../components/SwipeablePageContainer";
import Hackathons from "./Hackathons";

const Resume = () => {
	return (
		<SwipeablePageContainer>
			<Hackathons />
		</SwipeablePageContainer>
	);
};

export default Resume;
