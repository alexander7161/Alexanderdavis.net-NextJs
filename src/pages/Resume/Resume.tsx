import { Typography } from "@material-ui/core";
import React from "react";
import styled from "styled-components";
import { ContainerDiv } from "../../components";
import Hackathons from "./Hackathons";

const ResumeContainer = styled(ContainerDiv)`
	width: 100%;
	padding-top: 25vh;
	text-align: center;
`;

export default ({ theme }: { theme: any }) => {
	return (
		<ResumeContainer theme={theme}>
			<Hackathons />
			<Typography variant="h6" color="inherit">
				Coming Soon
			</Typography>
		</ResumeContainer>
	);
};
