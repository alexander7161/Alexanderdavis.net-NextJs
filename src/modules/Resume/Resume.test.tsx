import React from "react";
import { render } from "../../test-utils";
import Resume from "../Resume";

describe("Resume", () => {
	test("snapshot", () => {
		const { asFragment } = render(<Resume />);
		expect(asFragment()).toMatchSnapshot();
	});
});
