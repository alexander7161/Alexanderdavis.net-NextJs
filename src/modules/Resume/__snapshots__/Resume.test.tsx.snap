// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resume snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-bdnxRM bgBzlJ"
  >
    <div
      class="MuiContainer-root sc-gtsrHT ddRUIE MuiContainer-disableGutters MuiContainer-maxWidthMd"
    >
      <h5
        class="MuiTypography-root sc-hKFxyN eCWIEO MuiTypography-h5 MuiTypography-gutterBottom"
      >
        Hackathons (19 Total)
      </h5>
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-6"
        >
          <div
            class="MuiPaper-root MuiCard-root sc-dlnjwi bughy MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              class="MuiCardContent-root"
            >
              <div
                style="display: flex; justify-content: space-between; align-items: flex-start;"
              >
                <h2
                  class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
                >
                  Hack Zurich 2020
                </h2>
                <p
                  class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  mockDate
                </p>
              </div>
              <p
                class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
              >
                Zurich, Switzerland
              </p>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-6"
        >
          <div
            class="MuiPaper-root MuiCard-root sc-dlnjwi bughy MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              class="MuiCardContent-root"
            >
              <div
                style="display: flex; justify-content: space-between; align-items: flex-start;"
              >
                <h2
                  class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
                >
                  IC Hack 20
                </h2>
                <p
                  class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  mockDate
                </p>
              </div>
              <p
                class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
              >
                London, United Kingdom
              </p>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-6"
        >
          <div
            class="MuiPaper-root MuiCard-root sc-dlnjwi bughy MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              class="MuiCardContent-root"
            >
              <div
                style="display: flex; justify-content: space-between; align-items: flex-start;"
              >
                <h2
                  class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
                >
                  Junction
                </h2>
                <p
                  class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  mockDate
                </p>
              </div>
              <p
                class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
              >
                Helsinki, Finland
              </p>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-6"
        >
          <div
            class="MuiPaper-root MuiCard-root sc-dlnjwi bughy MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              class="MuiCardContent-root"
            >
              <div
                style="display: flex; justify-content: space-between; align-items: flex-start;"
              >
                <h2
                  class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
                >
                  Hack Zurich 2019
                </h2>
                <p
                  class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  mockDate
                </p>
              </div>
              <p
                class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
              >
                Zurich, Switzerland
              </p>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-6"
        >
          <div
            class="MuiPaper-root MuiCard-root sc-dlnjwi bughy MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              class="MuiCardContent-root"
            >
              <div
                style="display: flex; justify-content: space-between; align-items: flex-start;"
              >
                <h2
                  class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
                >
                  Everest Hack
                </h2>
                <p
                  class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  mockDate
                </p>
              </div>
              <p
                class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
              >
                Kathmandu, Nepal
              </p>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-6"
        >
          <div
            class="MuiPaper-root MuiCard-root sc-dlnjwi bughy MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              class="MuiCardContent-root"
            >
              <div
                style="display: flex; justify-content: space-between; align-items: flex-start;"
              >
                <h2
                  class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
                >
                  Infrahack
                </h2>
                <p
                  class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  mockDate
                </p>
              </div>
              <p
                class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
              >
                London, United Kingdom
              </p>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-6"
        >
          <div
            class="MuiPaper-root MuiCard-root sc-dlnjwi bughy MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              class="MuiCardContent-root"
            >
              <div
                style="display: flex; justify-content: space-between; align-items: flex-start;"
              >
                <h2
                  class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
                >
                  EU-FOSSA 2
                </h2>
                <p
                  class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  mockDate
                </p>
              </div>
              <p
                class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
              >
                Brussels, Belgium
              </p>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-6"
        >
          <div
            class="MuiPaper-root MuiCard-root sc-dlnjwi bughy MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              class="MuiCardContent-root"
            >
              <div
                style="display: flex; justify-content: space-between; align-items: flex-start;"
              >
                <h2
                  class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
                >
                  START Hack
                </h2>
                <p
                  class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  mockDate
                </p>
              </div>
              <p
                class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
              >
                St Gallen, Switzerland
              </p>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-6"
        >
          <div
            class="MuiPaper-root MuiCard-root sc-dlnjwi bughy MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              class="MuiCardContent-root"
            >
              <div
                style="display: flex; justify-content: space-between; align-items: flex-start;"
              >
                <h2
                  class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
                >
                  IC Hack 19
                </h2>
                <p
                  class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  mockDate
                </p>
              </div>
              <p
                class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
              >
                London, United Kingdom
              </p>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-6"
        >
          <div
            class="MuiPaper-root MuiCard-root sc-dlnjwi bughy MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              class="MuiCardContent-root"
            >
              <div
                style="display: flex; justify-content: space-between; align-items: flex-start;"
              >
                <h2
                  class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
                >
                  HackKing's 5.0
                </h2>
                <p
                  class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  mockDate
                </p>
              </div>
              <p
                class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
              >
                London, United Kingdom
              </p>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-6"
        >
          <div
            class="MuiPaper-root MuiCard-root sc-dlnjwi bughy MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              class="MuiCardContent-root"
            >
              <div
                style="display: flex; justify-content: space-between; align-items: flex-start;"
              >
                <h2
                  class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
                >
                  Junction
                </h2>
                <p
                  class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  mockDate
                </p>
              </div>
              <p
                class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
              >
                Helsinki, Finland
              </p>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-6"
        >
          <div
            class="MuiPaper-root MuiCard-root sc-dlnjwi bughy MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              class="MuiCardContent-root"
            >
              <div
                style="display: flex; justify-content: space-between; align-items: flex-start;"
              >
                <h2
                  class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
                >
                  Facemoji Keyjam
                </h2>
                <p
                  class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  mockDate
                </p>
              </div>
              <p
                class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
              >
                London, United Kingdom
              </p>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-6"
        >
          <div
            class="MuiPaper-root MuiCard-root sc-dlnjwi bughy MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              class="MuiCardContent-root"
            >
              <div
                style="display: flex; justify-content: space-between; align-items: flex-start;"
              >
                <h2
                  class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
                >
                  Jacobs Hack
                </h2>
                <p
                  class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  mockDate
                </p>
              </div>
              <p
                class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
              >
                Bremen, Germany
              </p>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-6"
        >
          <div
            class="MuiPaper-root MuiCard-root sc-dlnjwi bughy MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              class="MuiCardContent-root"
            >
              <div
                style="display: flex; justify-content: space-between; align-items: flex-start;"
              >
                <h2
                  class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
                >
                  HackUPC
                </h2>
                <p
                  class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  mockDate
                </p>
              </div>
              <p
                class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
              >
                Barcelona, Spain
              </p>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-6"
        >
          <div
            class="MuiPaper-root MuiCard-root sc-dlnjwi bughy MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              class="MuiCardContent-root"
            >
              <div
                style="display: flex; justify-content: space-between; align-items: flex-start;"
              >
                <h2
                  class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
                >
                  TechCrunch
                </h2>
                <p
                  class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  mockDate
                </p>
              </div>
              <p
                class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
              >
                Paris, France
              </p>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-6"
        >
          <div
            class="MuiPaper-root MuiCard-root sc-dlnjwi bughy MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              class="MuiCardContent-root"
            >
              <div
                style="display: flex; justify-content: space-between; align-items: flex-start;"
              >
                <h2
                  class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
                >
                  Copenhacks
                </h2>
                <p
                  class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  mockDate
                </p>
              </div>
              <p
                class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
              >
                Copenhagen, Denmark
              </p>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-6"
        >
          <div
            class="MuiPaper-root MuiCard-root sc-dlnjwi bughy MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              class="MuiCardContent-root"
            >
              <div
                style="display: flex; justify-content: space-between; align-items: flex-start;"
              >
                <h2
                  class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
                >
                  HEX
                </h2>
                <p
                  class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  mockDate
                </p>
              </div>
              <p
                class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
              >
                Eindhoven, The Netherlands
              </p>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-6"
        >
          <div
            class="MuiPaper-root MuiCard-root sc-dlnjwi bughy MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              class="MuiCardContent-root"
            >
              <div
                style="display: flex; justify-content: space-between; align-items: flex-start;"
              >
                <h2
                  class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
                >
                  Anvil Hack IV
                </h2>
                <p
                  class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  mockDate
                </p>
              </div>
              <p
                class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
              >
                London, United Kingdom
              </p>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-6"
        >
          <div
            class="MuiPaper-root MuiCard-root sc-dlnjwi bughy MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              class="MuiCardContent-root"
            >
              <div
                style="display: flex; justify-content: space-between; align-items: flex-start;"
              >
                <h2
                  class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
                >
                  HackCity
                </h2>
                <p
                  class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  mockDate
                </p>
              </div>
              <p
                class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
              >
                London, United Kingdom
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
